docs/codeql/conf.py

```diff
@@ -109,5 +109,5 @@ def setup(sphinx):
 # so a file named "default.css" will overwrite the builtin "default.css".
 html_static_path = ['_static']
 
-exclude_patterns = ['vale*', '_static', '_templates', 'codeql', 'learn-ql', 'reusables', 'images', 'support', 'ql-training', '_build', '*.py*', 'README.rst']
+exclude_patterns = ['vale*', '_static', '_templates', 'reusables', 'images', 'support', 'ql-training', 'query-help','_build', '*.py*', 'README.rst']
 ##############################################################################
```

Comment:
```suggestion
##############################################################################

```

Add a newline at the end of the file, just to keep the UNIX gods happy (or something like that) 🙈